client/rest-high-level/src/test/java/org/elasticsearch/client/MlTestStateCleaner.java

```diff
@@ -106,10 +107,28 @@ private void closeAllJobs() {
     }
 
     private void deleteAllDataFrameAnalytics() throws IOException {
+        stopAllDataFrameAnalytics();
```

Comment:
Do we need to stop DF analytics in `MlRestTestStateCleaner` as well?

Reply:
Good point. Will do.

Reply:
Done in https://github.com/elastic/elasticsearch/pull/56101/commits/5fab34a19b6fb62fc5fc35c1dcfd5073d9ebe884